npm/test-integration.js

```diff
@@ -51,7 +59,7 @@ module.exports = function (exit) {
 
             // run tests using the consolidated test set as a guide
             async.map(suites, function (test, next) {
-                console.info(` - ${test.name}`);
+                console.info(` - ${test.name.split('/')[2]}`);
```

Comment:
Interesting point. Tests have been arranged within folders, but that shouldn't be taken as a guarantee. I'll improve the logic here in the next commit.